src/tools/chip-cert/CertUtils.cpp

```diff
@@ -508,51 +508,52 @@ bool AddAuthorityKeyId(X509 * cert, X509 * caCert)
     return res;
 }
 
-bool ReadCertPEM(const char * fileName, X509 * cert)
-{
-    bool res    = true;
-    FILE * file = nullptr;
-
-    res = OpenFile(fileName, file);
-    VerifyTrueOrExit(res);
-
-    if (PEM_read_X509(file, &cert, nullptr, nullptr) == nullptr)
```

Comment:
How is PEM support preserved with this update?

Reply:
All various certificate formats are handled by the ReadCert() call. I believe there was some historical redundancy in this code...